server/tests/services/group.test.js

```diff
@@ -197,5 +197,51 @@ describe('# Group Service', () => {
           });
       });
     });
+
+    describe('# updateEscalationPolicy()', () => {
+      const userId = '123456789098765432123456';
+
+      let group;
+
+      beforeEach((done) => {
+        build('escalationPolicy', fixtures.escalationPolicy({ subscribers: [userId] }))
+          .then(escalationPolicy => build('group', fixtures.group({ escalationPolicy: escalationPolicy })))
+          .then((newGroup) => {
+            group = newGroup;
+            done();
+          });
+      });
+
+
+      it('should update a groups escalation policy', (done) => {
```

Comment:
100% a nitpick, but can you reformat so the grammar here is correct? (groups => group's)